src/components/Conditional.js

```diff
@@ -21,6 +21,8 @@ const Conditional = (props) => {
     isFalse: node.text === node.data.get('whenFalse'),
   };
 
+  const emptyConditionalFlag = node.text === conditional.whenTrue ? conditional.whenFalse : conditional.whenTrue;
```

Comment:
Unsure I follow this logic. Aren't we looking for if the conditional value is an empty string? So could it be

```js
const emptyConditionalFlag = conditional.value === '';
```

So in the props actually it could be:

```js
className: conditional.value ? 'conditional' : '';
```

Reply:
Actually ```conditional.value``` is undefined in that scope. ```node.text``` is being used to access the value of conditional. I have replaced this in the current commit. Please check!